lib/matplotlib/lines.py

```diff
@@ -534,13 +534,10 @@ def set_markevery(self, every):
 
         e.g., if `every=5`, every 5-th marker will be plotted.
 
-        ACCEPTS: [None | int | length-2 tuple of int | slice |
-        list/array of int | float | length-2 tuple of float]
-
         Parameters
         ----------
-        every: None | int | length-2 tuple of int | slice | list/array of int \
-| float | length-2 tuple of float
+        every: None or int or (int, int) or slice or List[int] or float or \
```

Comment:
Do we need the 'or' after everything in this list for some reason?  `one of: None, int, (int, int), slice, List[ind], float, or (float, float)` would be easier to read.   Or separate by semicolons?

Reply:
See http://www.sphinx-doc.org/en/master/usage/restructuredtext/domains.html#info-field-lists ("Multiple types in a type field will be linked automatically if separated by the word “or”").